concerned-coyotes/earlyinternet/utils/helpers.py

```diff
@@ -0,0 +1,10 @@
+def create_url_with_parameters(base_url: str, params: dict) -> str:
+    """Creates a url for the given base address with given parameters
+    as a query string."""
+
+    parameter_string = "&".join(f"{key}={value}"
+                                for key, value in params.items())
+    url = f"{base_url}?{parameter_string}"
+
+    # skip the last character as it is a &
+    return url[:-1]
```

Comment:
When using join the last character will not be the joined character. Therefore this will cut of the last character of the last `value`.